packages/pds/tests/auth.test.ts

```diff
@@ -0,0 +1,165 @@
+import AtpApi, { ServiceClient as AtpServiceClient } from '@atproto/api'
+import { SeedClient } from './seeds/client'
+import { CloseFn, runTestServer, TestServerInfo } from './_util'
+import * as locals from '../src/locals'
+
+describe('auth', () => {
+  let server: TestServerInfo
+  let client: AtpServiceClient
+  let close: CloseFn
+
+  beforeAll(async () => {
+    server = await runTestServer({
+      dbPostgresSchema: 'auth',
+    })
+    client = AtpApi.service(server.url)
+    close = server.close
+  })
+
+  afterAll(async () => {
+    await close()
+  })
+
+  const createAccount = async (info) => {
+    const { data } = await client.com.atproto.createAccount({}, info)
+    return data
+  }
+  const getSession = async (jwt) => {
+    const { data } = await client.com.atproto.getSession({}, undefined, {
+      headers: SeedClient.getHeaders(jwt),
+    })
+    return data
+  }
+  const createSession = async (info) => {
+    const { data } = await client.com.atproto.createSession({}, info)
+    return data
+  }
+  const deleteSession = async (jwt) => {
+    const { data } = await client.com.atproto.deleteSession({}, undefined, {
+      headers: SeedClient.getHeaders(jwt),
+    })
+    return data
+  }
+  const refreshSession = async (jwt) => {
+    const { data } = await client.com.atproto.refreshSession({}, undefined, {
+      headers: SeedClient.getHeaders(jwt),
+    })
+    return data
+  }
+
+  it('provides valid access and refresh token on account creation.', async () => {
+    const account = await createAccount({
+      username: 'alice.test',
+      email: 'alice@test.com',
+      password: 'password',
+    })
+    // Valid access token
+    const sessionInfo = await getSession(account.accessJwt)
+    expect(sessionInfo).toEqual({ did: account.did, name: account.username }) // @TODO standardize on name or username?
```

Comment:
I think this called-out a little inconsistency in the API: should we consider unifying on `name` or `username`?

Reply:
yeah I support 👍 

- "username" is the domain-style mutable id for a user. we should use this one
- "name" doesn't refer to anything in particular. could be displayName, username, pet name, first part of username before the host. confusing word that we should avoid in code
- "user" we've been using this to refer to `usernameOrDid`, but we may want a more descriptive term here cause it requires some context to understand

Reply:
Why username rather than handle?

@handle is clear signal that this is not meant as a real names policy and that a user could have many @handles that point to the same DID

Reply:
actually I really like this. handle is an increasingly common term that shouldn't be too disorienting for users. & it is much more descriptive than "username"

Reply:
updated 😁 
https://github.com/bluesky-social/atproto/issues/287